SurfaceDuoSamples/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  ~
+  -->
+
+<resources>
+    <string name="surface_duo_samples_app_name">Faltbare Apps von Surface Duo</string>
+    <string name="surface_duo_samples_app_title">Faltbare App-Beispiele von Surface Duo</string>
+    <string name="hello_to_the_app">Willkommen bei Foldable Samples von Surface Duo!</string>
+
+    <string name="drag_and_drop_info">Drag &amp; Drop - Zeigt, wie Sie ein Layout erstellen, in das Sie Inhalte aus anderen Apps ziehen und ablegen können.</string>
```

Comment:
This is in English, and the rest of the sample-names are in German, we have to go for an unified way, English or German but not mixed.